src/features/reviews/utils/adjustQueue.js

```diff
@@ -9,15 +9,35 @@
 
 import _ from 'lodash';
 
-import { MEANING, RADICAL } from 'src/common/constants';
+import { MEANING, RADICAL, RANDOM_ORDER, LOWEST_LEVEL_FIRST, CURRENT_LEVEL_FIRST, ASCENDING_SRS_STAGE, DESCENDING_SRS_STAGE } from 'src/common/constants';
 
 const ALLOWED_MAX_DISTANCE = 10;
 const ALLOWED_MIN_DISTANCE = 3;
 
-export default (queue, backToBackMode = false, meaningFirst = false) => {
+export default (queue, backToBackMode = false, meaningFirst = false, reviewOrder = RANDOM_ORDER) => {
+
   const newQueue = queue.slice();
   let currentIndex = 0;
 
+  // sort new queue based on the selected review order
```

Comment:
Can we add a `DEFAULT` option (selected by default) that doesn't apply any sorting at all?

Reply:
That was kinda my intention with the `RANDOM` option here, since it is how it works today -- albeit unintentionally.

I'm afraid that if we add a `DEFAULT` option that doesn't apply any sorting at all, the queue would inherit the last selected option. For example, if a user switches to `LOWEST_LEVEL_FIRST` and then back to the `DEFAULT` option, since the `DEFAULT` does nothing, the queue would still be sorted in the `LOWEST_LEVEL_FIRST` order. However if we keep the `RANDOM` as the default selection, we will keep the default, current behavior, although with an extra step.

Does that make sense?